### ISSUE TITLE
Terminal Preview and Terminal have the same icon in Alt-Tab and Win-Tab window switchers

### ISSUE THREAD

**Issue by metathinker** [microsoft/terminal#6777]
### Environment

**Windows build number:** Windows 10 ver. 2004 build 19041.329
**Windows Terminal version (if applicable):** 1.0.1811.0, Preview 1.1.1812.0

### Steps to reproduce

1. Open Windows Terminal.
1. Open Windows Terminal Preview.
1. Press Alternate-Tab or Windows-Tab.

### Expected behavior

The Terminal and Terminal Preview windows thumbnails have different icons displayed next to their titles.

### Actual behavior

The Terminal and Terminal Preview windows thumbnails have the same displayed next to their titles.
(Taskbar shows different icons as expected.)
![Windows-Tab screenshot showing Windows Terminal and Windows Terminal Preview with the same icons](https://user-images.githubusercontent.com/1039862/86488953-929ece80-bd17-11ea-857a-76f042185755.png)

**Comment by KalleOlaviNiemitalo** [comment]
> (Taskbar shows different icons as expected.)

That is true in two cases:

- If Windows Terminal Preview is pinned to the task bar and not running.
- If, in the taskbar settings, "Combine taskbar buttons" is "Always, hide labels"

However, if Windows Terminal Preview 1.1.1812.0 is running and "Combine taskbar buttons" is "Never", then the icon in the taskbar button lacks the blue preview marker.

I assume correcting the Windows Terminal Preview icon in Alt-Tab would fix the icon in the taskbar button as well, so I am not filing that as a separate issue. (Alternatively, <https://github.com/microsoft/terminal/issues/6556> would make the taskbar icon not depend on the application icon.)

**Comment by jtippet** [comment]
I believe there are two possible fixes:
* Fix the OS shell to prefer the MRT icon instead of preferring the win32 icon
* Add alternate versions of /res/terminal.ico

The 1st fix is clearly better, since it benefits any hybrid app.  But the 2nd fix is much easier, since it'd just take about an hour to gin up a new .ico file and hack the .RC file to refer to it when building the preview flavor.

**Comment by metathinker** [comment]
> * Fix the OS shell to prefer the MRT icon instead of preferring the win32 icon

For the benefit of people outside Microsoft or the Windows org:
"MRT" is the internal name for the Modern Resource Technology, or the [Resource Management System](https://docs.microsoft.com/windows/uwp/app-resources/) as it is referred to on docs.microsoft.com. It is the system for adding and loading text string, image, and file resources to a Universal Windows Platform app or other [MSIX-packaged](https://docs.microsoft.com/windows/msix/) app (including Desktop Bridge apps like Terminal), and for choosing the right resource to use depending on things like the user's language and system color choices, tile sizes, and the screen's scale factor.

Basically, if your MSIX-packaged desktop app's image resources are separate files or even separate MSIX packages, they may be loaded by MRT. If they're embedded in the .exe, they're the old-fashioned Win32 resources Mr. Tippet is referring to.

**Comment by DHowett** [comment]
So, I’m not averse to having a separate preview icon in the EXE file, but right now there are no compiled artifact differences between Release and Preview, only packaging and livery.
Is there something we could do by having all the resources in the executable and setting the window icon to one of them based on PFN?
